AzureVM.bicep

```
@@ -0,0 +1,208 @@
+    }        
+  }
+}
+
+resource shutdown_computevm_virtualMachineName 'Microsoft.DevTestLab/schedules@2018-09-15' = {
```

nit: the name has an extra `virtualMachineName` in it and, in general, for variables and resources Bicep prefers camel case and does some automagical things like word separation when rendering the Portal experience.

```suggestion
resource shutdownComputeVm 'Microsoft.DevTestLab/schedules@2018-09-15' = {
```